jina/parsers/__init__.py

```diff
@@ -5,6 +5,12 @@
 
 
 def set_pea_parser(parser=None):
+    """
+    Set the parser for the Pea
```

Comment:
Let's stick to the same standard when writing docstrings. Previously, we are following the standard sphinx way, https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html#the-sphinx-docstring-format

```suggestion
    """Set the parser for the Pea
```

Reply:
The linter doesn't complain about it (as we agreed to configure it), so unless someone always checks them manually, I don't think we should prioritize this.